app/src/main/java/org/oppia/app/walkthrough/WalkthroughActivity.kt

```diff
@@ -23,4 +23,8 @@ class WalkthroughActivity : InjectableAppCompatActivity() {
     activityComponent.inject(this)
     walkthroughActivityPresenter.handleOnCreate()
   }
+
+  override fun changeTo(pageNo: Int) {
```

Comment:
Replace `pageNo ` to `walkthroughPage` because directly we are passing a `WalkthroughPages` item.

Reply:
done

Reply:
@aggarwalpulkit596 This has been done incorrectly.
You have to change `pageNo` with `walkthroughPage ` and the function name should still be `changeTo`. Such that your final interface will look like this:
```
/** Listener for when an activity should change a fragment position. */
interface WalkthroughFragmentChangeListener {
  fun currentPage(walkthroughPage: Int)
}
```

Reply:
okay will do that right away